zapcli/zap_helper.py

```diff
@@ -394,5 +394,5 @@ def html_report(self, file_path):
     @staticmethod
     def _write_report(report, file_path):
         """Write report to the given file path."""
-        with open(file_path, 'w') as f:
+        with open(file_path, mode='wb') as f:
             f.write(report)
```

Comment:
Oh, I see. Anyway, this did not work on python 2.7: 
```
 UnicodeDecodeError: 'ascii' codec can't decode byte 0xef in position 37228: ordinal not in range(128)
```
I'll try to see if we can somehow bypass this, but if you want to play with it I am using this [repo](https://github.com/Soluto/webdriverio-zap-proxy) for testing.